dashboard/public/index.html

```diff
@@ -30,6 +30,16 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <!-- runtime endpoints config -->
+    <script type="text/javascript">
+      fetch("/api/v1/endpoints").then((response) => {
+        if (response.ok) {
+          response.text().then((data) => {
+            window.endpoints = JSON.parse(data);
+          });
+        }
```

Comment:
Should we be doing something if the response is _not_ OK?  (I don't want to leave our developers wondering what happened, if, for example, the express server didn't come up or something.)

Reply:
The nodeJS reflector isn't really the issue; yes, that's one *additional* failure mode in local npm developer mode, but, in theory, this could fail on a real server if the HTTP server (or reverse proxy) is down. Hmm. (And if it fails in dev mode because the nodeJS reflector wasn't started, it'll look just like an unreachable host anyway, because that's what it is.)

This code was copied directly from the old dashboard, and my changes *there* were directly descended from the original code to load the server configuration JSON file. However you're correct that there's no error handling here. (Which actually surprised me, because I'd thought there was. Huh.) I think this means we never had any error handling if the config file was missing or otherwise unavailable, I didn't notice and nobody pointed that out when I changed it. Intriguing.

I'm not quite sure what we could do except splash up an error message.

@MVarshini mentioned that she was interested in integrating the endpoints load into the Redux level rather than putting it out here in `index.html` (which has always made me a bit uncomfortable). If we can't load our UI app code, there's no point in worrying about the endpoints; and, if we can load the code and get it started, but can't load endpoints, we may be in a position at that level to at least generate a better looking error message. 😄 

If @MVarshini wants to take in this PR more or less "as is" instead of reworking it, I'll add an `else` that at least renders the HTTP response on the screen.

Reply:
I'm happy to defer to either you or @MVarshini.

I'm just imagining myself trying to run the dashboard locally...and, when I've screwed it up, I'm hoping that there will be helpful feedback....  😕